portfolio/src/main/java/com/google/sps/servlets/FavShowServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+@WebServlet("/show")
+public class FavShowServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      ArrayList<String> show = new ArrayList<String>();
+      show.add("Grey's Anatomy");
+      show.add("Modern Family");
+      show.add("Umbrella Academy");
+      show.add("Criminal Mind");
+      //response.getWriter().println(show);
```

Comment:
Prefer to remove this line of commented code.

Reply:
Done